src/psyclone/psyir/symbols/symboltable.py

```diff
@@ -903,14 +903,14 @@ def argument_datasymbols(self):
         return [sym for sym in self.datasymbols if sym.is_argument]
 
     @property
-    def global_symbols(self):
+    def import_symbols(self):
```

Comment:
Seeing this in use, I think `imported_symbols` would be a bit nicer.

Reply:
Done